requirements.txt

```diff
@@ -1,7 +1,8 @@
-boto3 ~= 1.14
+boto3 ~= 1.7.84; python_version < "3.10"
```

Comment:
It looks like Python 3.9 is end of life - we should drop support for 3.9 though we can do that in another ticket.

Reply:
I agree totally, I would actually pull this now from this file, it was so that snyk would resolve the proper versions fast enough to not time out.  But I discovered python 3.9's boto has urllib3 pinned < 2.  tl;dr had to update snyk to 3.10.  But I'll put a follow up to move it to 3.13 and clean this up.